pkg/endpoint/policy.go

```diff
@@ -605,7 +606,15 @@ func (e *Endpoint) regeneratePolicy(owner Owner, opts models.ConfigurationMap) (
 }
 
 // Called with e.Mutex UNlocked
-func (e *Endpoint) regenerate(owner Owner, reason string) error {
+func (e *Endpoint) regenerate(owner Owner, reason string) (retErr error) {
+	metrics.EndpointCountRegenerating.Inc()
+	defer func() {
+		metrics.EndpointCountRegenerating.Dec()
+		tags := map[bool]string{true: "succeeded", false: "failed"}
```

Comment:
Why not use a simply if statement?
```
if retErr != nil {
   metrics.CountEndpointsRegenerations.WithLabelValues("failed").Inc()
} else {
   metrics.CountEndpointsRegenerations.WithLabelValues("succeeded").Inc()
}
```
Do you need to set up a constant for `succeeded` and `failed` or not necessary?

Reply:
Good point. I made constants for the values... I'm not sure if I should also make constants for the field names...